code/frameworks/nextjs/src/preset.ts

```diff
@@ -108,6 +108,29 @@ export const babel: PresetProperty<'babel'> = async (baseConfig: TransformOption
     presets,
     babelrc: false,
     configFile: false,
+    overrides: [
```

Comment:
Shouldn’t we load `babelDefault` instead from the presets and apply it?

Reply:
I'm not sure what you mean with this, do you mean to somehow reuse the `babel` preset loaded in:
`code/core/src/core-server/presets/common-preset.ts`? 

So in theory `options.overrides` should actually contain that preset/override, but it gets ignored as the include regex doesn't match the filename in line 62. 

That is why I need to copy the override again.
I think we should solve it in:
https://github.com/storybookjs/storybook/issues/28467

But won't be easy I imagine.